api-reference/10 UI Components/dxPivotGrid/3 Methods/bindChart(chart_integrationOptions).md

```diff
@@ -51,6 +51,172 @@ Accepts one of the following values.
 - *"series"* - data fields go to the chart series.
 
```

Comment:
Can please make the following changes to the lines above?

Line 20:<br>
Specifies whether to alternate data fields in the resulting chart or not.<br>
`Specifies whether to alternate data fields in the resulting chart.`

Line 31:<br>
Accepts one of the following values.<br>
`Accepts one of the following values:`

Line 48:<br>
Accepts one of the following values.
`Accepts one of the following values:`

Line 50:<br>
`- *"args"* - data fields go to the chart arguments.`
What do you mean by "go to"? For example, do you mean "data fields are the chart arguments"?

Line 51:
`- *"series"* - data fields go to the chart series.`
See my comment above.

Reply:
Sounds good. Please apply your suggestion to lines 50 and 51.